Index.html

```diff
@@ -52,10 +52,192 @@ <h1>
         </section>
+
+        <section id="portfolio">
+         <article class="portfolio-work">
+               <img class="work-image" src="./assets//Snapshoot Portfolio.svg" alt="">
+            <div class="work-description">
+                <h1 class="work-title">Tonic</h1>
+                <ul class="job-description">
+                    <li>CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="work-technologie">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a  class="button">See Project</a>
+            </div>
+         </article>
+    
+         <article class="portfolio-work">
+            <img class="work-image" src="./assets/Snapshoot Portfolio 2.svg" alt="">
+        <div class="work-description">
+            <h1 class="work-title">Multi-Post <br>
+                Stories</h1>
+            <ul class="job-description">
+                <li>CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="work-technologie">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+            </ul>
+            <a class="button">See Project</a>
+        </div>
+     </article>
+
+     <article class="portfolio-work">
+        <img class="work-image" src="./assets/Snapshoot Portfolio 3.svg "alt="">
+    <div class="work-description">
+        <h1 class="work-title">Tonic</h1>
+        <ul class="job-description">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+        </ul>
+        <p>Tonic</p>
+        <ul class="work-technologie">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+        </ul>
+       <div> 
+        <a class="button">See Project</a>
+    </div>
+    </div>
+ </article>
+ <article class="portfolio-work">   
+    <img class="work-image" src="./assets/Snapshoot Portfolio 4.svg" alt="">
+<div class="work-description">
+    <h1 class="work-title">Multi-Post <br> Stories</h1>
+    <ul class="job-description">
+        <li>CANOPY</li>
+        <li>Back End Dev</li>
+        <li>2015</li>
+    </ul>
+    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="work-technologie">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+    </ul>
+    <a class="button">See Project</a>
+</div>
+</article>
+</section>
+
+<section id="portfolio">
+    <article class="portfolio-about">
+        <div class="about-h1">
+            <h1 class="about-h1">About <br> Myself</h1>
```

Comment:
- [Optional] I see that you have a lot of h1 elements in your document. Kindly note that each html document is supposed to have just one h1 element. This is best practice for SEO.